src/reducers/filter.js

```diff
@@ -0,0 +1,12 @@
+import { CHANGE_FILTER_VISIBILITY } from '../actions';
+
+const initialState = 'true';
```

Comment:
why is it a string and not just a boolean? what are other possible statuses? it would probably be even better if you list possible options here as a comment

Reply:
you're right, it should be boolean. it's just a mistake